src/output.h

```diff
@@ -402,6 +407,7 @@ inline void popup( const char *mes, Args &&... args )
 {
     popup( string_format( mes, std::forward<Args>( args )... ), PF_NONE );
 }
+void popup( input_event &evt, const std::string &text, PopupFlags flags );
```

Comment:
Because that would require including `input.h`, which is already dependency of many files, and would cause any file that is dependent on `output.h` to have dependency on `input.h`. Currently if you change `input.h`, a huge bunch of files are re-compiled, sometimes unnecessarily, I'd rather not make it worse.

Anyway, it's doubtful why the popup code was put in `output.h/cpp` in the first place, so I'll move it to a new file and make it return the input event instead.

Reply:
Ok, it turns out moving it to a new file would require a large set of changes. Even just changing it in `output.h/cpp` would conflict with one of my current PR #25810, so this'll have to wait until that one is merged.